filebeat/input/filestream/config.go

```diff
@@ -59,7 +59,7 @@ type stateChangeCloserConfig struct {
 	Renamed       bool          `config:"renamed"`
 }
 
-type readerConfig struct {
+type ReaderConfig struct {
```

Comment:
I see no reason why this struct must be exported? Is it for 'inlining' the settings in config?

If possible I would prefer to keep a packages interface (exported symbols) to a minimu. If the `config` was the problem we still can use:

```
type config struct {
  Reader readerConfig `config:",inline"`
  ...
}
```